### DISCUSSION TITLE
Automatic deployment Error

### DISCUSSION THREAD

**Discussion by yufelix1** [vercel/community#4415] [Help]
### Summary

09:12:52.045 | Done in 59.82s.
09:12:56.696 | Traced Next.js server files in: 4.612s
09:13:08.224 | Created all serverless functions in: 11.526s
09:13:08.356 | Collected static files (public/, static/, .next/static): 10.855ms
09:13:09.703 | Build Completed in /vercel/output [1m]
09:13:10.264 | Deploying outputs...
09:21:39.617 | Error: The Edge Functions build system is currently busy. Please try again later.

### Example

chat-gpt-next-h64h4ubu1-feiflown.vercel.app

### Steps to Reproduce

For several months, automatic deployment has been failing without success. but manual deployment can be successful. 
I need help, Thanks.

**Comment by mcsdevv** [comment]
Hey @feiflown, thanks for raising this, it definitely doesn't look right, sorry for the poor experience.

Let me check in with the team and come back to you.

**Reply by mcsdevv** [reply]
Following up, we're aware of the issue, apologies for the inconvenience!

We're aiming to roll a full fix for this within the coming weeks.

**Reply by yufelix1** [reply]
OK. thanks for the response

**Comment by mcsdevv** [comment]
Hey @feiflown thanks for your patience here!

I can confirm this has now been resolved and there should be no further issues with this. Appreciate you raising it!
